rcl/src/rcl/logging.c

```diff
@@ -95,7 +95,13 @@ rcl_logging_configure(const rcl_arguments_t * global_args, const rcl_allocator_t
   if (g_rcl_logging_ext_lib_enabled) {
     status = rcl_logging_external_initialize(config_file, g_logging_allocator);
     if (RCL_RET_OK == status) {
-      rcl_logging_external_set_logger_level(NULL, default_level);
+      // TODO(dirk-thomas) the return value should be typed and compared to
+      // constants instead of zero
```

Comment:
While this mismatch is unfortunately it is better then not enforcing to check the return value.

Reply:
@dirk-thomas mind to explain? `rcl_logging_external_set_logger_level()` returns an `rcl_ret_t` value.